server/src/db/migrations/20210201204619-create-user.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
```

Comment:
`primaryKey: true` makes the value required

Reply:
fixed